cost-analyzer/templates/cost-analyzer-ingestion-configmap.yaml

```
@@ -0,0 +1,12 @@
+{{- if .Values.kubecostProductConfigs.standardDiscount }}
```

```suggestion
{{- if (.Values.kubecostProductConfigs).standardDiscount }}
```

I believe you will need this to fix the Chart Build issue. "Only try to dereference standardDiscount if kubecostProductConfigs is defined".

---

Yeah oops, just saw that. My bad.